admin/load.php

```diff
@@ -531,6 +537,32 @@ function perflab_enqueue_modules_page_scripts() {
 	wp_enqueue_style( 'thickbox' );
 
 	wp_enqueue_script( 'plugin-install' );
+
+	// Bail early if module is not active.
+	$get_active_modules_with_standalone_plugins = perflab_get_active_modules_with_standalone_plugins();
+	if ( empty( $get_active_modules_with_standalone_plugins ) ) {
+		return;
+	}
+
+	wp_enqueue_script(
+		'perflab-module-migration-notice',
+		plugin_dir_url( __FILE__ ) . 'perflab-module-migration-notice.js',
+		array(),
+		'1.0.0',
+		array(
+			'strategy' => 'defer',
+		)
+	);
+
+	wp_localize_script(
+		'perflab-module-migration-notice',
+		'perflab_module_migration_notice',
+		array(
+			'ajaxurl'       => admin_url( 'admin-ajax.php' ),
+			'nonce'         => wp_create_nonce( 'perflab-install-activate-plugins' ),
+			'network_error' => esc_html__( 'Network response was not ok.', 'performance-lab' ),
```

Comment:
Since the string is not being printed here, it should not get escaped. The `wp_localize_script()` function will do the escaping.

```suggestion
			'network_error' => __( 'Network response was not ok.', 'performance-lab' ),
```